Content.Shared/Atmos/Piping/Binary/Systems/SharedGasValveSystem.cs

```diff
@@ -0,0 +1,69 @@
+using Content.Shared.Atmos.Piping.Binary.Components;
+using Content.Shared.Examine;
+using Content.Shared.Interaction;
+using Robust.Shared.Audio;
+using Robust.Shared.Audio.Systems;
+
+namespace Content.Shared.Atmos.Piping.Binary.Systems;
+
+public abstract class SharedGasValveSystem : EntitySystem
+{
+
+    [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<GasValveComponent, ComponentStartup>(OnInit);
+        SubscribeLocalEvent<GasValveComponent, ActivateInWorldEvent>(OnActivate);
+        SubscribeLocalEvent<GasValveComponent, ExaminedEvent>(OnExamined);
+    }
+
+    private void OnInit(EntityUid uid, GasValveComponent component, ComponentStartup args)
```

Comment:
```suggestion
    private void OnStartup(Entity<GasValveComponent> ent, ref ComponentStartup args)
```
naming convention, OnInit is for MapInitEvent